src/test/java/org/jabref/logic/layout/format/HTMLCharsTest.java

```diff
@@ -110,7 +106,7 @@ void newLine(String expected, String input) {
 
     @Test
     void quoteSingle() {
-        assertEquals("&#39;", layout.format("{\\textquotesingle}"));
+        assertEquals("&#39;", layout.format("\\textquotesingle"));
```

Comment:
The code change is OK. Reason: `{` is not used as parameter for a LaTeX command.

Reply:
Since the existing layouter is tested now, please add this back.